cartridge-cli.lua

```diff
@@ -1758,11 +1782,27 @@ local function construct_build_image_dockerfile()
         return nil, err
     end
 
+    local user_id, err = check_output('id -u')
+    if user_id == nil then
+        return nil, string.format('Failed to get user ID: %s', err)
+    end
+
+    user_id = user_id:strip()
+
+    local wrap_user_part = expand(
+        DOCKERFILE_WRAP_USER,
+        {
+            user_id = user_id,
+            username = os.getenv('USER') or 'myuser',
```

Comment:
Hm... Why "myuser", could we call `whoami` or `id -un` in this case?

Reply:
In fact, username It doesn't matter (only UID matters), but calling additional command can cause additional errors.